index.html

```diff
@@ -14,7 +14,7 @@
         <h1>Cool Rocks</h1>
     </header>
     <main>
-        <a href ="../products.html">Click here for cool rocks</a>
+        <a href="../products">Click here for cool rocks</a>
```

Comment:
I think you had this right the first time as products.html 
did you test this link to make sure it was working after making the change?